build.gradle

```diff
@@ -14,8 +14,8 @@ plugins {
     id 'maven-publish'
 }
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 1.7
```

Comment:
Presumed we wanted to enforce compilation to JDK 1.7. Let me know if not and I'll swap back.